web/pnpm-lock.yaml

```diff
@@ -1,5414 +0,0 @@
-lockfileVersion: '9.0'
```

Comment:
I think we should not drop this file.

Reply:
OK, I restore `pnpm-lock.yaml`

Reply:
You need to upgrade pnpm, so that this file will not be update.